pipelines/short-read-alignment-vector/SRA_Test1.json

```diff
@@ -0,0 +1,37 @@
+{
+  "ShortReadAlignment.sample_id": "AN0131-C",
+  "ShortReadAlignment.output_basename": "SRA_Test1",
+  "ShortReadAlignment.read_group_ids": [
```

Comment:
These were (manually, for now) pulled from the fastqs to put into the bam created by bwa mem as part of the RG string.  Not sure these are needed at all or if we only need the sample ID?

Reply:
Thanks - I'll update that.  I think that some of these values can be pulled from the fastq with a simple grep - but if these are the sort of fields that could be put directly into the input manifest (like the sample manifest), that would be far simpler and more reliable.